docs/source/kedro-viz_visualisation.md

```diff
@@ -194,6 +194,17 @@ The visualisation now includes the layers:
 
 ![](./images/pipeline_visualisation_with_layers.png)
 
+Duplicated definitions like:
+
+```yaml
+metadata:
+  kedro-viz:
+    layer: raw
+```
```

Comment:
can we also show the example here that you gave in the original issue on how to use it. thanks

Reply:
```yaml
_raw_layer: &raw_layer
  metadata:
    kedro-viz:
      layer: 01_raw
```

And then reuse it like this:

```yaml
companies:
  type: pandas.CSVDataset
  filepath: data/01_raw/companies.csv
  <<: *raw_layer

reviews:
  type: pandas.CSVDataset
  filepath: data/01_raw/reviews.csv
  <<: *raw_layer

Reply:
@rashidakanchwala yeah let me add this